commands/get/project.go

```diff
@@ -0,0 +1,111 @@
+package get
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"text/template"
+
+	"github.com/feedhenry/rhm/commands"
+	"github.com/feedhenry/rhm/storage"
+	"github.com/urfave/cli"
+)
+
+var projectTemplate = `
```

Comment:
Maybe we can have separate task to create `OutputFormatter` that would present output depending on the format user requesting by specifying `output` option. For example values can be: 
- `json` - raw json option, 
- `yaml` - we can use json2yaml converter, 
- `table` - we can use one of this libs
https://github.com/tatsushid/go-prettytable or https://github.com/crackcomm/go-clitable

What you think about it? Can we create new ticket?